build.js

```diff
@@ -55,15 +59,28 @@ process.nextTick(function () {
   var traceur    = require('traceur');
   var reacttools = require('react-tools');
   var tss        = require('typescript-simple');
+  var identity   = function (x) { return x; };
```

Comment:
Trivia: you could actually just use `String` - it also returns any string passed into it.